src/main/kotlin/io/polygon/kotlin/sdk/rest/PolygonRestClient.kt

```diff
@@ -122,6 +122,36 @@ constructor(
             requestIteratorFetch<TradesResponse>()
         )
 
+    /**
+     *  Get quotes for an equity/forex ticker within a range
+     *  Note that fx symbols are prepended with "C:" e.g. "C:EUR-USD"
+     *
+     * API Docs:
+     *     https://polygon.io/docs/stocks/get_v3_quotes__stockticker
+     *     https://polygon.io/docs/forex/get_v3_quotes__fxticker
+     */
```

Comment:
You can get quotes for options as well: https://polygon.io/docs/options/get_v3_quotes__optionsticker